mkosi/config.py

```diff
@@ -1411,6 +1413,7 @@ class Config:
     """
 
     profile: Optional[str]
+    config_paths: list[Path]
```

Comment:
I would name this `files`. The class is already named `Config`, no need to repeat it here.